components/movementsensor/rtkutils/ntrip.go

```diff
@@ -60,3 +92,113 @@ func NewNtripInfo(cfg *NtripConfig, logger logging.Logger) (*NtripInfo, error) {
 	logger.Debug("Returning n")
 	return n, nil
 }
+
+// ParseSourcetable gets the sourcetable and parses it.
+func (n *NtripInfo) ParseSourcetable() (*Sourcetable, error) {
+	reader, err := n.Client.GetSourcetable()
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := reader.Close(); err != nil {
+			log.Println("Error closing reader:", err)
```

Comment:
does this show up in the resource's logger? we can pass in the rdk logger if not.

Reply:
or return a wrapper error if you'd rather not use a logger.